litmus-portal/k8s-manifest.yml

```diff
@@ -69,7 +69,10 @@ spec:
               value: "litmus-portal@123"
             - name: EXTERNAL_ADDRESS
               value: ""
-            - name: SERVICE_ADDRESS
+            - name: NODE_NAME
```

Comment:
@gdsoumya I have set env variables here.

Reply:
It was missing before. it's good now

Reply:
Thank you @gdsoumya for your review.
Yaaa Sorry for that. That was my mistake.